### ISSUE TITLE
[FIX] Administration - > Users, no users in dropdown Characteristcs - Login

### ISSUE THREAD

**Issue by fgendorf** [glpi-project/glpi#1288]
Hi, the dropdown Characteristcs - Login in Administration User area, send to ajax/getDropdownUsers.php the values:

$_POST Array
(
    [all] => 0
    [right] => 
    [entity_restrict] => ["1"]
    [searchText] => 
    [page_limit] => 100
    [page] => 1
)

In file ajax/getDropdownUsers.php have the code:

```php
if (!isset($_POST['right'])) {
   $_POST['right'] = "all";
}
```

but $_POST['right'] is set with no value, so the code don't put "all" on it.

to fix change code to:

```php
if (empty($_POST['right'])) {
   $_POST['right'] = "all";
}
```

In that way, if $_POST['right'] don't exists or is empty, the value will be filled by "all".

**Comment by orthagh** [comment]
Hello @fgendorf 

Ok for adding a new test like:
```php
if (!isset($_POST['right']) || empty($_POST['right'])) {
   $_POST['right'] = "all";
}
```

i tagged this issue for next bugfixes release.

btw, i can't replicate emptiness of right param:
![image](https://cloud.githubusercontent.com/assets/418844/20271736/04cff2f6-aa8c-11e6-9531-3592e73d12d4.png)

Could you precise me how to trigger this issue precisely ?

**Comment by fgendorf** [comment]
Hi, I don't know why, but in my installation, the $_POST["right"] is empty on Administration -> Users
![screenshot_2016-11-14_14-23-52](https://cloud.githubusercontent.com/assets/15034336/20272863/6a9af1be-aa76-11e6-8c37-0e3bb0517602.png)

**Comment by fgendorf** [comment]
Hi, I updated to 9.1.1 and found the bug, in file ajax/searchoptionvalue.php

change:

```php
$options2['right'] = $searchopt['right'];
```
to
```php
$options2['right'] = (!empty($searchopt['right'])?$searchopt['right']:'all');
```

**Comment by trasher** [comment]
Seems the same issue as #1394; fixed in #1499
